### PR TITLE
Removed mpich and slurm components from schizo framework

### PR DESCRIPTION
**Pull request by Michael-uofl**
This pull request removes the `mpich` and `slurm` components from the `schizo` framework.

### CODE REVIEW

File: src/tools/Makefile.am [hunk 1]
```diff
@@ -28,21 +28,14 @@
 SUBDIRS += \
 	tools/prted \
 	tools/prun \
-	tools/pcc \
```
Comment (jsquyres) [inline src/tools/Makefile.am:31]: I think we need to re-evaluate what `pcc` is used for, and whether it should actually be removed or not.

**Review by jsquyres — CHANGES_REQUESTED** [review]

File: src/tools/Makefile.am (hunk 1 above)
Comment (hppritcha) [reply src/tools/Makefile.am:31]: I think we should not remove pcc, we just don't want to document it.

Comment (rhc54) [reply src/tools/Makefile.am:31]: Yeah, removal would be problematic in general. You could switch to "pmixcc", but then you'd need to modify all test and example compiles to point to the PMIx location and deal with those complexities (e.g., what is the user's PATH when they want to build an example).

**Comment by rhc54** [comment]
Am I misunderstanding things? I keep seeing additional PR's popping up that are also labeled as "remove mpich and slurm components"

**Pull request closed without merging**